pyquil/api/_quantum_computer.py

```diff
@@ -654,7 +654,13 @@ def get_qc(name: str, *, as_qvm: bool = None, noisy: bool = None,
 
 
 @contextmanager
-def local_qvm() -> Iterator[Tuple[subprocess.Popen, subprocess.Popen]]:
+def local_runtime(
```

Comment:
I think `local_forest_runtime` is a bit more descriptive

Reply:
Agreed.